_data/governments.yml

```diff
@@ -337,6 +337,9 @@ Italy:
   - RegioneER
   - TeamDigitale
 
+Jamaica:
+  - Bertland Hope
```

Comment:
That doesn't appear to be a GitHub organization

Reply:
Agree, and if @bertlandh was intended, that also isn't an org (it's a user) and it doesn't appear to be for a government organization. Please feel free to reopen or otherwise further comment if this is incorrect!